### PR TITLE
Adapt js "to-code" to handle the multi cell selection 

### PR DESCRIPTION
**Pull request by parleur**
For now, only handle cells conversion to codecells. Not perfect with the main menu toolbar.

### CODE REVIEW

File: notebook/static/notebook/js/notebook.js [hunk 1]
```diff
@@ -1176,7 +1176,17 @@ define(function (require) {
         var len = this.ncells();
         return this.insert_cell_below(type,len-1);
     };
-
+    
+
```
Comment (captainsafia) [inline notebook/static/notebook/js/notebook.js]: Can you add a documentation string before the function to describe what it does? You can look at the documentation string above `to_code` for guidance.

**Comment by captainsafia** [comment]
Great PR! Thanks for submitting. I made a quick note about adding documentation.

**Comment by minrk** [comment]
ping @ellisonbg, since this appears to be an action not doing the right thing in multi-select context.

**Comment by Carreau** [comment]
That's part of the various actions that don't apply (yet) to multiple selection (see #738 )

**Comment by Carreau** [comment]
closes #802

**Comment by parente** [comment]
The commits here went in with #809. This can be closed.

**Pull request closed without merging**